messages/en.json

```diff
@@ -94,5 +94,9 @@
   "store/reviews.list.sortOptions.highestRated": "Highest Rated",
   "store/reviews.list.sortOptions.lowestRated": "Lowest Rated",
   "store/reviews.list.anonymous": "Anonymous",
-  "store/reviews.list.pagination.textOf": "of"
+  "store/reviews.list.pagination.textOf": "of",
+  "store/reviews.list.graph.stars": "{value} stars",
```

Comment:
You can use pluralization feature of ICU format

```
{value, plural, =0{0 stars} =1{1 star} other{# stars}}
```

You can read more here:
https://format-message.github.io/icu-message-format-for-translators/